### ISSUE TITLE
[C3M5_C] Various issues

### ISSUE THREAD

**Issue by hawkwind3** [Realm667/WolfenDoom#1010]
C3M5_C ...

- [x] 1. Line 3572 needs a tex on it.
- [x] 2. Small glitch in the floor of sectors 1378/1393 after the gate has opened. Compare sectors 1613/1614 where this does not occur.
- [x] 3. Note that the floors of sectors 1475 and 1484 are different. And ceilings of said sectors could use a slight alignment. Might be worth comparing with the other side. Sectors 1545/1578.
- [x] 4. Can partly clip through all 4 big guns. T112 for example.

:x: 5. Before parachuting out, the ss uniform is back at sector 1329.
> The uniform is put on the rack by Ascher -- N00b 24 Apr 2023
- [x] 6. Can still escape even if the crash timer has reached zero.

**Comment by hawkwind3** [comment]
7. Action on line 8119 cannot be used until the engines have been sabotaged. When BJ returns here, the switch is magically fixed, without a reason given. I suggest that, just before parachuting out, Asher says something like " I also found a repair kit and fixed the damaged switch while waiting for you"
8. BJ can get trapped at x 12544, y -18672. Maybe a small crate here or remove sector 2440.
9.  Check the sides of the captains bathtub at sector 1679.
10.  A bunch of texture offset fixes ...
Sidedefs 14897, 14756, 14645, 14761, 14897,  14909, 14267, 14255, 14288, 14420, 14326 all need an X offset of 32. Sidedef 14358 needs an X offset of 0 and sidedef 14363 needs an X offset of -24. Applies to texture CRAT_GR2.
Sidedefs 6567 and 7907 need an X offset of 7 and sidedefs 6573 and 7619 need an X offset of -6.
11. Sector 1784 and within has a fog like effect. Maybe due to a Fade setting ? Looks odd, unless the effect is to simulate cigarette smoke ...

**Comment by x3003** [comment]
>10. A bunch of texture offset fixes ...

I think you mean these ones ?
Screenshots taken w/ the latest git version / gzdoom 4.8 pre 10.

![Screenshot_Doom_20211001_172505](https://user-images.githubusercontent.com/77861029/135646705-b7766424-490d-44d6-9284-39644d391ae5.png)

![Screenshot_Doom_20211001_173040](https://user-images.githubusercontent.com/77861029/135647183-bc79010e-9b8e-434b-9386-7b552d160504.png)

**Comment by hawkwind3** [comment]
Yes. These are the fixes ...
Sidedefs 14897, 14756, 14645, 14761, 14897, 14909, 14267, 14255, 14288, 14420, 14326 all need an X offset of 32. Sidedef 14358 needs an X offset of 0 and sidedef 14363 needs an X offset of -24. Applies to texture CRAT_GR2.

**Comment by hawkwind3** [comment]
- [ ] 13. When lift sector 420 is rising, reaches target floor height, or lowering, the ceiling texture changes 90 degrees, and looks unscaled.

**Comment by Username-N00b-is-not-available** [comment]
Fixed problems spotted by @hawkwind3.
When collecting some of the three necessary items (repair kit or parachutes) Ascher will message you via radio even if you have not met him in the bar yet. And when you later go to him, the objectives will not be shown as completed. I tried to experiment with checking them in the same script that gives you the objectives (after ```NamedScriptWait```ing on ```boaobjectivesset```), but it wouldn't work, so I scrapped it. I guess it is not that much of an issue.
I had to add a ```KillEverything``` ZScript helper function for use on the map.
